woodwork/statistics_utils.py

```diff
@@ -23,8 +23,10 @@ def _get_describe_dict(dataframe, include=None, callback=None):
             will be returned.
         callback (callable, optional): function to be called with incremental updates. Has the following parameters:
 
-            - update: percentage change (float between 0 and 100) in progress since last call
-            - progress_percent: percentage (float between 0 and 100) of total computation completed
+            - update (int/float): change in progress since last call
```

Comment:
Is there a way to make this consistently return one type? Or is this `int/float` for a reason?

Reply:
I pulled that directly from the original issue, but I the current implementation will only return `int` values. I'm thinking in general we could have progress measured in non whole number increments, but that might be unlikely. 

I think we could restrict to just `int` for the time being, but if we ever had function in which progress was calculated as a `float` (like percentage for, example) we would just need to remember to update this or cast the values to int before calling the callback function.

Reply:
That's fair. Let's change the doc string to `int` to accurately reflect the returned value. 

We could also cast to `float` to better description the progress (rather than `int`) when the situation you describe comes up.

Reply:
Sounds good.

Reply:
@gsheni Aside from this change was there anything else you saw that needs to be updated?

Reply:
nope, looks good